src/utils/ParseRouteUtils.js

```diff
@@ -65,7 +66,25 @@ function mergeDirections(first, second): Object {
   const stops = first.stops.concat(second.stops);
   const tripIDs = first.tripIdentifiers.concat(second.tripIdentifiers);
 
+  // The combined delay is only the first bus route's delay, because that is
+  // how much the combined route is initially delayed by and is what the user
+  // needs to know in order to board the route on time.
+  const delay = isNullOrUndefined(first.delay) ? 0 : first.delay;
```

Comment:
Should the delay be 0 or nil if `first.delay` is undefined? Does 0 signify on time?

Reply:
oh wait that's a good point thanks